packages/manager/cypress/e2e/core/kubernetes/lke-update.spec.ts

```diff
@@ -1341,3 +1350,606 @@ describe('LKE cluster updates', () => {
+          'revision-id': mockRevisionId,
+          addresses: { ipv4: ['10.0.0.0/24'], ipv6: undefined },
+        }
+      );
+      const mockControlPaneACL = kubernetesControlPlaneACLFactory.build({
+        acl: mockACLOptions,
+      });
+      const mockUpdatedControlPlaneACL1 = kubernetesControlPlaneACLFactory.build(
+        {
+          acl: mockUpdatedACLOptions1,
+        }
+      );
+
+      mockGetCluster(mockCluster).as('getCluster');
+      mockGetControlPlaneACL(mockCluster.id, mockControlPaneACL).as(
+        'getControlPlaneACL'
+      );
+      mockUpdateControlPlaneACL(mockCluster.id, mockUpdatedControlPlaneACL1).as(
+        'updateControlPlaneACL'
+      );
+
+      cy.visitWithLogin(`/kubernetes/clusters/${mockCluster.id}`);
+      cy.wait(['@getAccount', '@getCluster', '@getControlPlaneACL']);
+
+      // confirm summary panel
+      cy.contains('Control Plane ACL').should('be.visible');
+      ui.button
+        .findByTitle('Enable')
+        .should('be.visible')
+        .should('be.enabled')
+        .click();
+
+      ui.drawer
+        .findByTitle('Control Plane ACL')
+        .should('be.visible')
+        .within(() => {
+          // Confirm submit button is disabled if form has not been changed
+          ui.button
+            .findByTitle('Update')
+            .should('be.visible')
+            .should('not.be.enabled');
+
+          cy.contains(
+            "Control Plane ACL secures network access to your LKE cluster's control plane. Use this form to enable or disable the ACL on your LKE cluster, update the list of allowed IP addresses, and adjust other settings."
+          ).should('be.visible');
+
+          // confirm Activation Status section and toggle on 'Enable'
+          cy.contains('Activation Status').should('be.visible');
+          cy.contains(
+            'Enable or disable the Control Plane ACL. If the ACL is not enabled, any public IP address can be used to access your control plane. Once enabled, all network access is denied except for the IP addresses and CIDR ranges defined on the ACL.'
+          ).should('be.visible');
+          cy.findByText('Enable Control Plane ACL');
+          ui.toggle
+            .find()
+            .should('have.attr', 'data-qa-toggle', 'false')
+            .should('be.visible')
+            .click();
+
+          // confirm submit button is now enabled
+          ui.button
+            .findByTitle('Update')
+            .should('be.visible')
+            .should('be.enabled');
+
+          // confirm Revision ID section and edit Revision ID
+          cy.findAllByText('Revision ID').should('have.length', 2);
+          cy.contains(
+            'A unique identifing string for this particular revision to the ACL, used by clients to track events related to ACL update requests and enforcement. This defaults to a randomly generated string but can be edited if you prefer to specify your own string to use for tracking this change.'
```

Comment:
```suggestion
            'A unique identifying string for this particular revision to the ACL, used by clients to track events related to ACL update requests and enforcement. This defaults to a randomly generated string but can be edited if you prefer to specify your own string to use for tracking this change.'
```

This looks like a typo. @coliu-akamai Can we change this in the actual code and update these tests accordingly?

Reply:
eagle eyes! tysm for catching 🙏